web/src/actions/auth.js

```diff
@@ -38,7 +51,7 @@ export const login = (username, password) => dispatch => {
     )
     .then(req => {
       dispatch(completeLogin(req.data));
-      dispatch(fetchApd());
+      dispatch(loadData(req.data.activities));
```

Comment:
Also do this data loading dance when the user finishes logging in, in case they weren't already.